src/components/propTypes/cardTypes.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+export const cardTypes = {
```

Comment:
give this variable a more explicit name - like MovieCardPropType (so it's obvious to which element it is related)